api/capi/src/nnstreamer-capi-single.c

```diff
@@ -338,9 +299,7 @@ ml_single_set_gst_info (ml_single * single_h, const ml_tensors_info_h info)
     case ML_NNFW_TYPE_CUSTOM_FILTER:
       ml_tensors_info_copy_from_ml (&gst_in_info, info);
 
-      klass = g_type_class_peek (G_TYPE_TENSOR_FILTER_SINGLE);
-      if (klass == NULL
-          || klass->set_input_info (single_h->filter, &gst_in_info,
+      if (single_h->klass->set_input_info (single_h->filter, &gst_in_info,
```

Comment:
In previous invoke() case, logical negation (i.e. !) is used instead of comparing `FALSE`. (160 line) However, comparing `FALSE` is used in this case. It would be better when keeping consistency.

Reply:
Updated.